src/EventListener/WriteListener.php

```diff
@@ -75,7 +75,7 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
                 $hasOutput = true;
                 if (null !== $this->resourceMetadataFactory) {
                     $resourceMetadata = $this->resourceMetadataFactory->create($attributes['resource_class']);
-                    $hasOutput = false !== $resourceMetadata->getOperationAttribute($attributes, 'output_class', null, true);
+                    $hasOutput = null !== $resourceMetadata->getOperationAttribute($attributes, 'output', ['class' => $attributes['resource_class']], true)['class'];
```

Comment:
The `class` key may not be defined

Reply:
nope because if it's `null` it defaults to `$attributes['resource_class']`